server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraSessionModule.java

```diff
@@ -243,6 +253,43 @@ private AsyncRetryExecutor provideAsyncRetryExecutor(ScheduledExecutorService sc
         return new AsyncRetryExecutor(scheduler);
     }
 
+    @VisibleForTesting
+    @Provides
+    @Singleton
+    CassandraConfiguration provideCassandraConfiguration(CassandraSessionConfiguration sessionConfiguration) throws ConfigurationException {
+        PropertiesConfiguration propertiesConfiguration = sessionConfiguration.getConfiguration();
+
+        return CassandraConfiguration.builder()
```

Comment:
We already have a similar discussion and I do not remember the conclusion of it but here there is some logic duplication.
This is the builder responsability to choose for the default value not for this provideCassandraConfiguration.